controller/src/main/java/io/pravega/controller/server/rest/ModelHelper.java

```diff
@@ -67,9 +66,6 @@ public static final StreamConfiguration getCreateStreamConfig(final CreateStream
                     retentionPolicy = getRetentionPolicy(createStreamRequest.getRetentionPolicy().getTimeBasedRetention(),
```

Comment:
this should also take max and set it. same for size based.

Reply:
Fixed